[doc] Add diagnostic tool section to GCP Dataproc getting started page [skip ci] (#6755)
NVIDIA/spark-rapids: docs/get-started/getting-started-gcp.md

+--------------------------------+--------------------------------------------------+--------------------------------------------------------------------------------------------------+
```

## Diagnose GPU Cluster

> Diagnostic tool can be run to check GPU cluster with Spark RAPIDS environment is healthy and

Author: refined

+-------------------------------+----------------------+----------------------+

+-----------------------------------------------------------------------------+
| Processes:                                                                  |
|  GPU   GI   CI        PID   Type   Process name                  GPU Memory |
|        ID   ID                                                   Usage      |
|=============================================================================|
|  No running processes found                                                 |
+-----------------------------------------------------------------------------+
NVRM version: NVIDIA UNIX x86_64 Kernel Module  460.106.00  Tue Sep 28 12:05:58 UTC 2021
GCC version:  gcc version 7.5.0 (Ubuntu 7.5.0-3ubuntu1~18.04)
Connection to 35.188.25.57 closed.
*** Running diagnostic function "cuda_version" ***
Connection to 34.171.177.95 closed.
found cuda major version: 11
*** Running diagnostic function "cuda_version" ***
Connection to 35.188.25.57 closed.
found cuda major version: 11
*** Running diagnostic function "rapids_jar" ***
Connection to 34.171.177.95 closed.
found rapids jar version: 2.12-22.06.0

> ...

Reviewer: That's a good idea. Let's discuss in the requirement doc.

> ...

Author: refined based on new requirement

```

> Please note, diagnostic tool supported

Author: refined

+--------------------------------+--------------------------------------------------+--------------------------------------------------------------------------------------------------+
```

## Diagnose GPU Cluster

The diagnostic tool can be run to check a GPU cluster with RAPIDS Accelerator for Apache Spark

> is healthy and ready for Spark jobs, such as installed Nvidia driver version, cuda-toolkit, RAPIDS

Reviewer 2: ```suggestion
is healthy and ready for Spark jobs, such as checking the version of installed NVIDIA driver version, cuda-toolkit, RAPIDS
```

Author: refined

> Accelerator and run some Spark test jobs etc. This tool also can be used by the frontline support

Reviewer 2: ```suggestion
Accelerator and running Spark test jobs etc. This tool also can be used by the frontline support
```

Author: refined

+-----------------------------------------------------------------------------+
| Processes:                                                                  |
|  GPU   GI   CI        PID   Type   Process name                  GPU Memory |
|        ID   ID                                                   Usage      |
|=============================================================================|
|  No running processes found                                                 |
+-----------------------------------------------------------------------------+
NVRM version: NVIDIA UNIX x86_64 Kernel Module  460.106.00  Tue Sep 28 12:05:58 UTC 2021
GCC version:  gcc version 7.5.0 (Ubuntu 7.5.0-3ubuntu1~18.04)
Connection to 35.188.25.57 closed.
*** Running diagnostic function "cuda_version" ***
Connection to 34.171.177.95 closed.
found cuda major version: 11
*** Running diagnostic function "cuda_version" ***
Connection to 35.188.25.57 closed.
found cuda major version: 11
*** Running diagnostic function "rapids_jar" ***
Connection to 34.171.177.95 closed.
found rapids jar version: 2.12-22.06.0
...
```

Please note that the diagnostic tool supports the following:

> * Dataproc 2.0 with image of Debian 10 or Ubuntu 18.04 (Rocky8 is coming soon)

Reviewer 2: ```suggestion
* Dataproc 2.0 with image of Debian 10 or Ubuntu 18.04 (Rocky8 support is coming soon)
```

Author: refined
